test/runner.js

```diff
@@ -278,48 +278,92 @@ describe('Runner', () => {
 
       assert.equal(
-        'The field `Query.viewer` is missing a description.',
+        'The field `User.id` is missing a description.',
         errors[3].message
       );
-      assert.equal(7, errors[3].location.line);
+      assert.equal(10, errors[3].location.line);
       assert.equal(
         `${__dirname}/fixtures/schema/user.graphql`,
         errors[3].location.file
       );
       assert.equal(errors[3].rule, 'fields-have-descriptions');
 
       assert.equal(
-        'The field `Comment.body` is missing a description.',
+        'The field `User.username` is missing a description.',
         errors[4].message
       );
-      assert.equal(2, errors[4].location.line);
+      assert.equal(11, errors[4].location.line);
       assert.equal(
-        `${__dirname}/fixtures/schema/comment.graphql`,
+        `${__dirname}/fixtures/schema/user.graphql`,
         errors[4].location.file
       );
       assert.equal(errors[4].rule, 'fields-have-descriptions');
 
       assert.equal(
-        'The field `Comment.author` is missing a description.',
+        'The field `User.email` is missing a description.',
         errors[5].message
       );
-      assert.equal(3, errors[5].location.line);
+      assert.equal(12, errors[5].location.line);
       assert.equal(
-        `${__dirname}/fixtures/schema/comment.graphql`,
+        `${__dirname}/fixtures/schema/user.graphql`,
         errors[5].location.file
       );
       assert.equal(errors[5].rule, 'fields-have-descriptions');
+
+      assert.equal(
+        'The field `OldUser.id` is missing a description.',
+        errors[6].message
+      );
+      assert.equal(16, errors[6].location.line);
+      assert.equal(
+        `${__dirname}/fixtures/schema/user.graphql`,
+        errors[6].location.file
+      );
+      assert.equal(errors[6].rule, 'fields-have-descriptions');
+
+      assert.equal(
+        'The field `Query.viewer` is missing a description.',
+        errors[7].message
+      );
+      assert.equal(20, errors[7].location.line);
+      assert.equal(
+        `${__dirname}/fixtures/schema/user.graphql`,
+        errors[7].location.file
+      );
+      assert.equal(errors[7].rule, 'fields-have-descriptions');
+
+      assert.equal(
+        'The field `Comment.body` is missing a description.',
+        errors[8].message
+      );
+      assert.equal(2, errors[8].location.line);
+      assert.equal(
+        `${__dirname}/fixtures/schema/comment.graphql`,
+        errors[8].location.file
+      );
+      assert.equal(errors[8].rule, 'fields-have-descriptions');
+
+      assert.equal(
+        'The field `Comment.author` is missing a description.',
+        errors[9].message
+      );
+      assert.equal(3, errors[9].location.line);
+      assert.equal(
+        `${__dirname}/fixtures/schema/comment.graphql`,
+        errors[9].location.file
+      );
+      assert.equal(errors[9].rule, 'fields-have-descriptions');
```

Comment:
I think a migration towards Jest and snapshots would save a lot of maintenance time. 😄

Reply:
Can you elaborate on this? 😄 I'm not super familiar with Jest and snapshots.